### PR TITLE
Make User Page show Reviews and Bookshelf instead of Messages

### PR DESCRIPTION
**Pull request by karenfrilya97**

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -190,7 +190,7 @@ public User getUser(String email) {
   public List<Review> getReviewsByUser(String user) {
     Query query =
             new Query("Review")
-                    .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/codeu/data/Datastore.java]: Good catch! Was this not working before or is it just not used?

File: src/main/webapp/js/book-list.js [hunk 2]
```diff
@@ -58,7 +58,11 @@ function buildReadingStatusButton(userBook, readingStatus) {
           status: readingStatus
         },
         success: () => {
+<<<<<<< HEAD
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/book-list.js]: Looks like some stuff left behind from merging with master?

File: src/main/webapp/js/user-bookshelf.js [hunk 3]
```diff
@@ -111,7 +110,8 @@ function buildReadingStatusOption(userBook, readingStatus) {
         status: readingStatus
       },
       success: () => {
-        window.location = `/user-bookshelf.html?user=${userBook.user}`;
+        window.location = `/user-page.html?user=${userBook.user}#user-book-container`;
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/user-bookshelf.js]: Would the `#user-book-container` make the browser always scroll to the bookshelf?

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (karenfrilya97) [reply src/main/java/com/google/codeu/data/Datastore.java]: It wasn't used before.

I think @archthegit used it to try showing a list of reviews per user when the About Book page was not implemented yet. But it wasn't used anymore after About Book was implemented.

File: src/main/webapp/js/book-list.js (hunk 2 above)
Comment (karenfrilya97) [reply src/main/webapp/js/book-list.js]: Oops yeah I missed it.

File: src/main/webapp/js/user-bookshelf.js (hunk 3 above)
Comment (karenfrilya97) [reply src/main/webapp/js/user-bookshelf.js]: Hmm this line just redirects to the bookshelf tab in the user page. If the user gets to this page from the link in the navigation bar, it won't scroll to the bookshelf. Does that answer your question?

**Comment by karenfrilya97** [comment]
I recorded a video of the new user page UI here [https://drive.google.com/file/d/1Uxfvj0qWFHxQbbO8ZL8tUin3qqs7KJcF/view](url)

**Pull request merged**